### PR TITLE
Ahodges/video overview

### PR DESCRIPTION
**Pull request by lamagnifica**
@rjgrubb, @jhendersonedx, and @WatsonEmily, I moved information out of several existing chapters to create the overview in video_overview.rst. I reorganized somewhat and added the flowcharts. I also added the naming convention section to the video_uploads.rst file. Please let me know if I've introduced errors.
@mhoeber , @catong , @srpearce , I applied style guidelines that we agreed on last week. 

HTML and PDF available here: http://draft-video.readthedocs.org/en/latest/

### CODE REVIEW

File: en_us/video/source/video_overview.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+After these services are set up, course teams use Studio to upload one file to
+the edX servers for each of the videos that they want to include in their
+course. For more information, see :ref:`Uploading Videos in Studio`.
+
+************************************
+Video Encoding and Hosting Overview
+************************************
+
+After a video file uploads to the edX servers successfully, the automated
```
Comment (mhoeber) [inline en_us/video/source/video_overview.rst]: uploads is awkward.  "After you upload the video file . . .

File: en_us/video/source/video_overview.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ :alt: Flowchart of course team uploading a video, followed by edX assigning a
+     video ID and then transcoding it into four formats and transferring the
+     results to two host sites
+
+The automated encoding and hosting process takes 24 hours to complete. If a
+step fails to complete successfully the process includes multiple automated
+retries.
+
+Course teams can track the status of each video file on the Studio **Video
+Uploads** page as the videos upload to edX and then go through automated
```
Comment (mhoeber) [inline en_us/video/source/video_overview.rst]: can delete "upload to edX and then "

File: en_us/video/source/video_overview.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Videos to a Course`.
+
+.. image:: Images/add_video_process.png
+ :alt: Flowchart of course team uploading a video, followed by edX assigning a
+     video ID and then transcoding it into four formats and transferring the
+     results to two host sites
+
+When a learner uses the edX video player or the edX mobile app to access a
+course video, the video ID is what provides the connection to the URLs that
+host each differently-encoded versions of the original video file.
```
Comment (mhoeber) [inline en_us/video/source/video_overview.rst]: version (singular) ?

**Comment by mhoeber** [comment]
a few small comments, thanks.  :+1:

**Comment by jhendersonedx** [comment]
Very clear. The tricky part about process is that the report page is separate from the place where the edX ID must be added. Should we add a note that suggests authors either have 2 instances of Studio open (one to the Outline/Video component and one for the video upload report), or download the entire report so it's easy to have the edX ID handy?

**Comment by lamagnifica** [comment]
@lou-wang , @nasthagiri , adding you as an FYI.

**Comment by lamagnifica** [comment]
@jhendersonedx added to video_course.rst

**Pull request merged**